### PR TITLE
chore: minor cleanup

### PR DESCRIPTION
**Pull request by ShmuelLotman**
Closes #

Minor css changes and hotkey to one that is less cluttered

### CODE REVIEW

File: src/me/components/DocSearchWidget.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ const DocSearchWidget: FC = () => {
   return (
     <div className="WidgetSearch">
       <DocSearch />
-      <p className="WidgetHelperText">Press CTRL + h on any page to search</p>
+      <p className="WidgetHelperText">Press CTRL + M on any page to search</p>
```
Comment (Asalem1) [inline src/me/components/DocSearchWidget.tsx:33]: not sure if it makes a difference but `m` vs `M`?

**Review by Asalem1 — APPROVED** [review]
just a thought, maybe have the key be a variable somewhere in case this changes from `m` to `p` tomorrow

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 51 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2198)

File: src/me/components/DocSearchWidget.tsx (hunk 1 above)
Comment (ShmuelLotman) [reply src/me/components/DocSearchWidget.tsx:33]: Yeah, @mavarius specified it should be an uppercase M

**Pull request merged**